bin/packages/build.js

```diff
@@ -63,10 +63,12 @@ const isScssFile = ( filepath ) => {
  * @return {string}             Build path
  */
 function getBuildPath( file, buildFolder ) {
-	const pkgName = getPackageName( file );
+	// if the file has extension of ts, replace with js
+	const fileName = file.replace( '.ts', '.js' );
```

Comment:
We still build TS as before but they need to be built as JS files.

Reply:
I'm not sure what you mean? Compilation of ts files with straight babel doesn't rename the file that gets generated, so this renames any file thats `ts` to `js` so that webpack doesn't try recompile it.

Reply:
I see, sorry I mis-understood, makes total sense then :)